.dprintrc.json

```diff
@@ -0,0 +1,35 @@
+{
+  "$schema": "https://dprint.dev/schemas/v0.json",
+  "projectType": "openSource",
+  "lineWidth": 80,
+  "indentWidth": 2,
+  "typescript": {
+    "deno": true,
+    // todo: remove
+    "parameters.trailingCommas": "never",
+    "arguments.trailingCommas": "never",
+    "typeParameters.trailingCommas": "never"
```

Comment:
It would be good to do an initial PR or commit with these off, then a final commit turning these on. There's too much noise with them on at the start.

Reply:
SGTM

but is there any way we can avoid having adding this top-level  .dprintrc.json? Can it be tools/dprint.json ?

Reply:
For the best experience the CLI and code editor plugins automatically picks up the config file if it's at either:

* `.dprintrc.json`
* `config/.dprintrc.json`

Doing this also means that running `dprint fmt` in any descendant directory will cause it to use that config file. Perhaps we could move all these config files to a `config` directory? Is that a standard thing to do? I'd be open to adding more directories here if it's "standard" across projects, but I don't think `tools` is (adding it wouldn't be a big deal though).

That said, it is possible to move the configuration file and manually specify it via `--config` or `-c`, but I wouldn't recommend it because then it's not as nice of an experience—doesn't work with code editor plugins at the moment (will require some manual setup or vscode config somewhere if ever supported) and means `--config` needs to be supplied every time a command is run. I'm also on the fence about removing `--config`.